src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -790,6 +788,29 @@ bool CommandRequestImpl::IsResultCodeUnsupported(
          (first.is_unsupported_resource && second.is_unsupported_resource);
 }
 
+std::string GetComponentNameFromInterface(const HmiInterfaces::InterfaceID& interface) {
+    switch(interface) {
+    case HmiInterfaces::HMI_INTERFACE_Buttons:
+        return "Buttons";
+    case HmiInterfaces::HMI_INTERFACE_BasicCommunication:
+        return "BasicCommunication";
+    case HmiInterfaces::HMI_INTERFACE_VR:
+        return "VR";
+    case HmiInterfaces::HMI_INTERFACE_TTS:
+        return "TTS";
+    case HmiInterfaces::HMI_INTERFACE_UI:
+        return "UI";
+    case HmiInterfaces::HMI_INTERFACE_Navigation:
+        return "Navigation";
+    case HmiInterfaces::HMI_INTERFACE_VehicleInfo:
+        return "VehicleInfo";
+    case HmiInterfaces::HMI_INTERFACE_SDL:
+        return "SDL";
+    default:
+        return "Unknown type";
+    }
+}
+
 void CommandRequestImpl::AddTimeOutComponentInfoToMessage(
```

Comment:
@pold500 I am still thinking that this is the right way to fix the issue, did you give it a try?